<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1549</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1751</width>
      <height>901</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="Children">
     <widget class="QPushButton" name="Tim">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>171</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Tim</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Lieke">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>60</y>
        <width>171</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Lieke</string>
      </property>
     </widget>
     <widget class="QPushButton" name="kind3">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>60</y>
        <width>171</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="kind4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>171</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="kind5">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>210</y>
        <width>171</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="kind6">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>210</y>
        <width>171</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Pages">
     <widget class="QPushButton" name="page1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>191</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Pagina 1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="page2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>90</y>
        <width>191</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Pagina 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="page3">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>90</y>
        <width>191</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Pagina 3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="page4">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>90</y>
        <width>191</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Pagina 4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="page5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>240</y>
        <width>191</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Pagina 5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="page6">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>240</y>
        <width>191</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Pagina 6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="page7">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>240</y>
        <width>191</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Pagina 7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="page8">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>240</y>
        <width>191</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Pagina 8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pagex">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>390</y>
        <width>191</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Pagina ...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Pageview">
     <widget class="QLabel" name="screen">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>1361</width>
        <height>491</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/fakevideo/video.PNG&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="previous_page">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>610</y>
        <width>221</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Vorige Pagina</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next_page">
      <property name="geometry">
       <rect>
        <x>1270</x>
        <y>620</y>
        <width>221</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Volgende Pagina</string>
      </property>
     </widget>
     <widget class="QPushButton" name="replay">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>530</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Opnieuw</string>
      </property>
     </widget>
     <widget class="QLabel" name="text">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>620</y>
        <width>871</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;-------------------------------------------------------------------&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;-----------------------------------------------------------------------------------&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;------------------------------------------------------------------&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="pagenr">
      <property name="geometry">
       <rect>
        <x>1454</x>
        <y>530</y>
        <width>21</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Books">
     <widget class="QPushButton" name="book1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>451</width>
        <height>601</height>
       </rect>
      </property>
      <property name="text">
       <string>Milo en Lana op de kinderboerderij</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1549</width>
     <height>26</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionKinderen"/>
    <addaction name="actionBoeken"/>
    <addaction name="actionPaginas"/>
    <addaction name="separator"/>
    <addaction name="actionInstellingen"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionKinderen">
   <property name="text">
    <string>Kinderen</string>
   </property>
  </action>
  <action name="actionBoeken">
   <property name="text">
    <string>Boeken</string>
   </property>
  </action>
  <action name="actionPaginas">
   <property name="text">
    <string>Pagina's van huidige boek</string>
   </property>
  </action>
  <action name="actionInstellingen">
   <property name="text">
    <string>Instellingen</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
